recipes/data-management/quantumleap/docker-compose.yml

```diff
@@ -0,0 +1,129 @@
+version: '3.3'
+
+services:
+
+  quantumleap:
```

Comment:
there is no deploy description, the recipe should not only cover crate ha, but also quantumleap api.

Reply:
right, I forgot about this.